packages/babel-helper-member-expression-to-functions/src/index.js

```diff
@@ -362,6 +372,16 @@ const handle = {
 
     // MEMBER?.(ARGS) -> _optionalCall(MEMBER, ARGS)
     if (parentPath.isOptionalCallExpression({ callee: node })) {
+      // Replace `function (a, x = a.b.#c?.()) {}` to `function (a, x = (() => a.b.#c?.())() ){}`
+      // so the temporary variable can be injected in correct scope
+      // This can be further optimized to avoid unecessary IIFE
+      if (scope.path.isPattern()) {
```

Comment:
Is it worth extracting this into a reusable function since it's the same, for maintenance purposes?

Reply:
I slightly prefer to leave it as-is: Because the same code snippet is used in only two places in this file.